src/test/suite/fixtures/recorded/subtoken/takeFirstChar5.yml

```diff
@@ -16,6 +16,6 @@ initialState:
 finalState:
   documentContents: //**
   selections:
-    - anchor: {line: 0, character: 0}
-      active: {line: 0, character: 1}
+    - anchor: {line: 0, character: 2}
```

Comment:
@pokey Notice that we now select the token to the right that will change current behavior

Reply:
The problem is that `/\p{L}\p{M}*|\P{L}/gu` matches whitespace and new lines as well